### PR TITLE
Solves even more bugs

### PR DESCRIPTION
**Pull request by Roguim**
### Ticketed bug solves:
- `DARK-37` solved
- `DARK-36` solved
- `DARK-35` solved
- `DARK-30` solved
- `DARK-13` solved
- `DARK-12` should be solved
### Solved bugs that had no ticket:
- Underage/other join course error, probably used elsewhere
- Increased compatibility with modern schoology layout
**_Fix for `DARK-34` somewhat queued up, but commented out since it had some major contrast issues, and I want to get these bugs fixed ASAP._**

### CODE REVIEW

File: css/modern.css [hunk 1]
```diff
@@ -263,6 +336,21 @@ fieldset div div div label span {
     background: var(--background-color);
 }
 
+._pendo-step-container-styles {
+    background-color: var(--accent) !important;
+}
+.pendo-mock-flexbox-element:nth-child(2)>button {
```
Comment (aopell) [inline css/modern.css:342]: Does this selector have to be this specific? It's okay if it does, but would it break things if it were changed to say `pendo-mock-flexbox-element > button` or even `pendo-mock-flexbox-element button`?

**Review by aopell — COMMENTED** [review]
Looks pretty good! See my one comment, mostly just curious about it, no pressure if you can't change it.

I might remove the commented code before release, but otherwise great! Thank you!

File: css/modern.css (hunk 1 above)
Comment (Roguim) [reply css/modern.css:342]: iirc that was for the pendo box in 266e0b2, that was for the second one only since only the second one needed to be changed.

Comment (aopell) [reply css/modern.css:342]: Got it. Makes sense.

**Review by aopell — APPROVED** [review]

**Pull request merged**